runtime/src/main/jni/v8_inspector/src/inspector/v8-network-agent-impl.cpp

```diff
@@ -0,0 +1,74 @@
+//
+// Created by pkanev on 2/22/2017.
+//
+
+#include <assert.h>
+#include "v8-network-agent-impl.h"
+
+namespace v8_inspector {
+
+namespace NetworkAgentState {
+static const char networkEnabled[] = "networkEnabled";
```

Comment:
Indentation is according to the formatting tool used - astyle. https://github.com/NativeScript/android-runtime/blob/master/tools/astyle.js#L11